bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1290 @@
+			<parameter name="slot" type="integer" min="1" max="256">
+				<label>Slot</label>
+				<description>Slot</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="t66">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>T66</label>
+		<description>Current (0, 25) A</description>
+		<channels>
+			<channel id="value" typeId="float6n"/>
+			<channel id="lastStatusStored" typeId="lastStatusStored"/>
+			<channel id="healthy" typeId="healthy"/>
+		</channels>
+		<config-description>
+			<parameter name="node" type="integer" min="1" max="256">
+				<label>Node</label>
+				<description>Node</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="slot" type="integer" min="1" max="256">
+				<label>Slot</label>
+				<description>Slot</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="t67">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>T67</label>
+		<description>Power (0, 6500) W</description>
+		<channels>
+			<channel id="value" typeId="float6n"/>
+			<channel id="lastStatusStored" typeId="lastStatusStored"/>
+			<channel id="healthy" typeId="healthy"/>
+		</channels>
+		<config-description>
+			<parameter name="node" type="integer" min="1" max="256">
+				<label>Node</label>
+				<description>Node</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="slot" type="integer" min="1" max="256">
+				<label>Slot</label>
+				<description>Slot</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="t68">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>T68</label>
+		<description>Pressure measure (0, 1500) hPa</description>
+		<channels>
+			<channel id="value" typeId="float6n"/>
+			<channel id="lastStatusStored" typeId="lastStatusStored"/>
+			<channel id="healthy" typeId="healthy"/>
+		</channels>
+		<config-description>
+			<parameter name="node" type="integer" min="1" max="256">
+				<label>Node</label>
+				<description>Node</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="slot" type="integer" min="1" max="256">
+				<label>Slot</label>
+				<description>Slot</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="topic">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>Action Message</label>
+		<description>Look at: Souliss Wiki, Peer 2 Peer Communication. That are messages published in broadcast from souliss
+			nodes in channels defined by two value: number and variant.</description>
+		<channels>
+			<channel id="value" typeId="float6n"/>
```

Comment:
This looks more like a trigger channel, than a stateful channel.

Reply:
so what should I do?

Reply:
You could make it of kind `trigger`, if you agree that this channel has no actual state, but is triggered when an event occurs. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#trigger-channel-types

Reply:
changed to trigger type